### PR TITLE
New package: Intuit.QuickBooksDesktopAccountant2023 version R6

### PR DESCRIPTION
**Pull request by jahvari**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/126925)

### CODE REVIEW

**Comment by jahvari** [comment]
Closes #126844

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

(The installer is behind the console window.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 10:18:48 AM:  > 2023-11-17 10:18:47.489 [CLI ] Package hash verification failed. SHA256 in manifest 
[74574be8027627617e230a5b145b456cbd7faadceece682a51590f6c54571320] does not match download 
[b8da53359f4143f5bbc374b617dab73e3cd4547ed6f492bca95a69206635fcc5]

File: manifests/i/Intuit/QuickBooksDesktopAccountant2023/R6/Intuit.QuickBooksDesktopAccountant2023.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: Intuit.QuickBooksDesktopAccountant2023
+PackageVersion: R6
+InstallerType: exe
+Scope: machine
+ElevationRequirement: elevatesSelf
+Installers:
+- Architecture: x64
+  InstallerUrl: https://dlm2.download.intuit.com/akdlm/SBD/QuickBooks/2023/Latest/QuickBooksPremier2023.exe
+  InstallerSha256: 74574BE8027627617E230A5B145B456CBD7FAADCEECE682A51590F6C54571320
```
Comment (stephengillie) [inline manifests/i/Intuit/QuickBooksDesktopAccountant2023/R6/Intuit.QuickBooksDesktopAccountant2023.installer.yaml:12]:
```suggestion
  InstallerSha256: B8DA53359F4143F5BBC374B617DAB73E3CD4547ED6F492BCA95A69206635FCC5
```

**Comment by jahvari** [comment]
@stephengillie Does the installer have to allow and support silent installation? This specific installer will require a license and product key to be entered.

**Comment by stephengillie** [comment]
> @stephengillie Does the installer have to allow and support silent installation? This specific installer will require a license and product key to be entered before the install can proceed.

Hi @jahvari,

This is still a requirement.

**Pull request closed without merging**